components/common/AsyncActionHandler.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
`common` has a specific meaning in chromium and this should not be at the top level of components like this